app/scripts/lib/typed-message-manager.js

```diff
@@ -123,8 +123,16 @@ export default class TypedMessageManager extends EventEmitter {
     if (req) {
       msgParams.origin = req.origin;
     }
-    this.validateParams(msgParams);
-
+    let warning;
+    try {
+      this.validateParams(msgParams);
```

Comment:
Rather than include this validation here then explicitly suppress it, maybe it would be simpler to perform the validation in the UI? At least until we graduate this from a warning to a hard error.

Reply:
👀